sample/src/NimblePros.SampleToDo.Core/ContributorAggregate/Contributor.cs

```diff
@@ -5,15 +5,21 @@ namespace NimblePros.SampleToDo.Core.ContributorAggregate;
 
 public class Contributor : EntityBase, IAggregateRoot
 {
-  public string Name { get; private set; }
+  public string Name { get; private set; } = default!;
 
   public Contributor(string name)
   {
-    Name = Guard.Against.NullOrEmpty(name, nameof(name));
+    SetName(name);
   }
 
   public void UpdateName(string newName)
   {
-    Name = Guard.Against.NullOrEmpty(newName, nameof(newName));
+    SetName(newName);
+  }
+
+  public Contributor SetName(string newName)
```

Comment:
When would we use a fluent builder for entities?

Reply:
1- For setting multiple properties at the same time.
2- Preventing multiple overloaded methods